index.js

```diff
@@ -318,7 +318,9 @@ Tree.prototype._join = function (data) {
     }
   }
 
+  this.overflowClassed(data.length)
```

Comment:
I think it would be good to leave overflow out of the name.  That speaks more to how things work behind the scenes than to how a caller should interact with it.  I think `onresize` is fine, but I prefer just `resize`.  I think of it as an instruction to handle the tree container having been resized.